kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/CheckpointV2ReadSuite.scala

```diff
@@ -181,65 +181,6 @@ class CheckpointV2ReadSuite extends AnyFunSuite with TestUtils with ExpressionTe
     }
   }
 
-  test("v2 checkpoint support with an empty sidecar") {
```

Comment:
Removing this test as it replaces the side car file, but the manifest file contains previous side file size. This size when passed to the FileIO causes EOF exceptions in the Parquet reader.

This case is a rare edge case (where sidecar generated are zero file size). It is not possible to generate one with the delta-spark. For the parquet reader it doesn't matter if the file has zero or more records.